scipy/stats/_stats_py.py

```diff
@@ -6771,7 +6771,10 @@ def _permutation_ttest(a, b, permutations, axis=0, equal_var=True,
 
     # Calculate the p-values
     cmps = compare[alternative](t_stat, t_stat_observed)
-    pvalues = cmps.sum(axis=0) / permutations
+    if n_max > permutations:
+        pvalues = (cmps.sum(axis=0) + 1) / (permutations + 1)
+    else:
+        pvalues = cmps.sum(axis=0) / permutations
```

Comment:
Considered:
```suggestion
    # Randomized test p-value calculation should use biased estimate; see e.g.
    # https://www.degruyter.com/document/doi/10.2202/1544-6115.1585/
    adjustment = 1 if n_max > permutations else 0
    pvalues = (cmps.sum(axis=0) + adjustment) / (permutations + adjustment)
```
but feel free to leave it alone.